lib/inc/cpp-pcp-client/connector/connector.hpp

```diff
@@ -141,7 +146,7 @@ class LIBCPP_PCP_CLIENT_EXPORT Connector {
     /// It is safe to call monitorConnection in a multithreading
     /// context; it will return as soon as the dtor has been invoked.
     ///
-    /// Throw a connection_fatal_error if, in case of dropped
+    /// Throws a connection_fatal_error if, in case of dropped
```

Comment:
Exceptions will be stored by the monitoring task and propagated by a subsequent `stopMonitoring` call. I'll correct that.